src/config/config.rs

```diff
@@ -125,11 +144,14 @@ impl Config {
                 girs_dirs.push(config_dir.join(dir));
             }
         }
-        let girs_version = girs_dirs
+        let mut girs_version = girs_dirs
             .iter()
-            .filter_map(repo_hash)
-            .next()
-            .unwrap_or_else(|| "???".into());
+            .map(|d| GirVersion {
+                gir_dir: d.clone(),
+                hash: repo_hash(d),
```

Comment:
Probably for a separate PR: can we make this return `None` if `d` is not a git repository (or simply a directory within the root repository)?

Reply:
For now I'll apply the current rule: "???". But please open an issue if you want it to changed.